code/modules/clothing/masks/smokable.dm

```diff
@@ -201,6 +201,9 @@
 	ignitermes = "<span class='notice'>USER fiddles with FLAME, and manages to light their NAME.</span>"
 	brand = "\improper Trans-Stellar Duty-free"
 
+/obj/item/clothing/mask/smokable/cigarette/can_be_injected_by(var/atom/injector)
+	return TRUE
```

Comment:
Uh, why should this differ from `/atom/movable/can_be_injected_by(var/atom/injector)`? Now this returns true regardless of adjacency or free space. (Free space is checked in the one place `can_be_injected_by()` is called, so if it's unnecessary in the `/atom/movable` override, it should probably be removed from there.)